ui/src/translations/en.json

```diff
@@ -1171,6 +1171,7 @@
     "chart": "Chart",
     "chart preview": "Chart preview",
     "no_code": {
+      "togglePebble": "Toggle Pebble editor",
```

Comment:
Small nitpick - maybe make this be a snake case, to align with the rest of the relatively new stuff (spaces in the old keys are 🤢).